src/js/control-bar/text-track-controls/descriptions-button.js

```diff
@@ -0,0 +1,25 @@
+import CaptionsButton from './captions-button.js';
+
+/**
+ * The button component for toggling and selecting descriptions
+ *
+ * @constructor
+ */
+class DescriptionsButton extends CaptionsButton {
```

Comment:
Interesting. Do captions and descriptions have independent styles or do they share them?